README.md

```diff
@@ -1,42 +1,599 @@
+---
+layout: bt_wiki
+title: NSX Plugin
+category: Plugins
+draft: false
+weight: 300
+
+plugin_link: https://raw.githubusercontent.com/cloudify-cosmo/cloudify-nsx-plugin/master/plugin.yaml
+---
+
 # cloudify-nsx-plugin
 Cloudify Network Virtualization with VMware (NSX) plugin
 
-# Check platform example
-## Linux example
+# Common Documentation
+
+# Description
+
+The VMWare NSX plugin allows users to use a vCenter NSX based infrastructure for deploying services(both of network and secure) and applications.
```

Comment:
I think the part in parentheses should probably be:
(both network and security)